### PR TITLE
Adding rapids

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rapids/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a2a98d9a52aea3039603bb822382755ac418bb68612a12404a9d5fedc8d66edf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/rapids/meta.yaml:14]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/rapids/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - pip
+    - pyramid
+    - python
+    - pyyaml
+    - setuptools
+    - venusian
+    - zope.interface
```
Comment (marcelotrevisani) [inline recipes/rapids/meta.yaml:24]:
```suggestion
    - pip
    - python
```

File: recipes/rapids/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (synapticarbors) [inline recipes/rapids/LICENSE.txt:1]: Please delete this manually packaged license file. It's included in the source distribution from pypi

File: recipes/rapids/meta.yaml (hunk 1 above)
Comment (Ryan-Dmello) [reply recipes/rapids/meta.yaml:14]: Made suggested change

File: recipes/rapids/meta.yaml (hunk 2 above)
Comment (Ryan-Dmello) [reply recipes/rapids/meta.yaml]: Made suggested change

File: recipes/rapids/LICENSE.txt (hunk 3 above)
Comment (Ryan-Dmello) [reply recipes/rapids/LICENSE.txt]: Removed license file as suggested

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin, please ping team

**Pull request merged**